### PR TITLE
Fixes #19860 - Fix Style/For cop

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by mmoll** [comment]
For comparsion, enforcing the "for" style would lead to 838 changes (as opposed to 19 now). As always, vote :+1: and :-1: here.

**Comment by mmoll** [comment]
[test upgrade]

File: lib/tasks/model.rake [hunk 1]
```diff
@@ -44,21 +44,23 @@ def consolidate(mappings, dryrun)
       puts "There is a problem with the entry with regular expression #{rex}"
       next
     end
+    # rubocop:disable Lint/UselessAssignment
     original_models = Model.count
+    # rubocop:enable Lint/UselessAssignment
```
Comment (tbrisker) [inline lib/tasks/model.rake]: I think this cop is actually correct, this line should be outside the loop

**Comment by mmoll** [comment]
[test foreman]

File: lib/tasks/model.rake (hunk 1 above)
Comment (mmoll) [reply lib/tasks/model.rake]: Thanks, should be all good now :bowtie:

**Review by ekohl — APPROVED** [review]

**Review by tbrisker — COMMENTED** [review]
Looks like enough +1 and no -1, thanks @mmoll !

**Review by tbrisker — APPROVED** [review]

**Pull request merged**